AnnService/inc/Core/Common/Dataset.h

```diff
@@ -200,6 +207,16 @@ namespace SPTAG
                 return true;
             }
 
+            bool Refine(const std::vector<SizeType>& indices, Dataset<T>& data)
+            {
+                SizeType R = (SizeType)(indices.size());
+                data.Initialize(R, cols);
+                for (SizeType i = 0; i < R; i++) {
+                    std::memcpy((void*)data.At(i), (void*)At(indices[i]), sizeof(T) * cols);
```

Comment:
what's the relationship between typename T and SizeType here?

Reply:
SizeType is used to represent the number of vectors. T is the vector data type.